elasticsearch/pom.xml

```diff
@@ -54,6 +54,31 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
I don't understand what your question relates to :)
But I didn't add anything new to the local settings, I did rebuild ES from sources locally though.

How else are you testing with the right ES version? the one compatible with Lucene 5.2 is not released yet..